appeals/api/package.json

```diff
@@ -21,7 +21,7 @@
     "lint:js": "eslint .",
     "lint:js:fix": "eslint . --fix",
     "clean": "rimraf .turbo node_modules dist",
-    "swagger-autogen": "node ./src/server/swagger.js && npx prettier ./src/server/swagger-output.json --write",
+    "swagger-autogen": "node ./src/server/swagger.js && npx prettier ./src/server/openapi.json --write",
```

Comment:
I think we should rename this to `gen-api-spec` or something, but we can do that separately to both applications and appeals.